pages/release-notes.mdx

```diff
@@ -48,11 +51,11 @@ So altering your plans will no longer impact the other models and fields ouputs.
 
 It unfortunately means that your generated data output will change with this release.
 
-#### 🚨 Alias strategy is now opt-in
+#### 🛠️ Alias strategy is now opt-in
```

Comment:
We should use the emoji 🚨 for breaking changes, and this is one.

Reply:
Noted